activesupport/lib/active_support/callbacks.rb

```diff
@@ -675,10 +675,17 @@ def set_callback(name, *filter_list, &block)
         # <tt>:unless</tt> options may be passed in order to control when the
         # callback is skipped.
         #
-        #   class Writer < Person
-        #      skip_callback :validate, :before, :check_membership, if: -> { age > 18 }
+        #   class Writer < PersonRecord
+        #      skip_callback :save, :before, :saving_message
```

Comment:
Why remove the if option described above?

https://github.com/rails/rails/blob/30b69f83691d45f02b911fd4e40132c68ab3a054/activesupport/lib/active_support/callbacks.rb#L674-L676

Reply:
Oops, it was removed accidentally.